homework/home2.1/tests_cpp.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
there are to issues here:
1. You can encapsulate this check of the output using the function - it will make the code shorter and more readable.
2. There is no need to have a separate scripts for c++ and java programs. Actually it should be the same program from the functionality perspective. The only thing is the name of binary to run which can be handled by setting bash variable to `./main` or `java Main`